src/ui/drawing/ISurface.hh

```diff
@@ -108,12 +102,12 @@ public:
     /// <summary>
     /// Creates a new offscren surface.
     /// </summary>
-    virtual std::unique_ptr<ISurface> CreateSurface(int nWidth, int nHeight) const = 0;
+    virtual std::unique_ptr<ISurface> CreateSurface(const Size& nWH) const = 0;
 
     /// <summary>
     /// Creates a new offscren surface with an alpha channel.
     /// </summary>
-    virtual std::unique_ptr<ISurface> CreateTransparentSurface(int nWidth, int nHeight) const = 0;
+    virtual std::unique_ptr<ISurface> CreateTransparentSurface(const Size& nWH) const = 0;
```

Comment:
It was to reduce number of parameters needed. Not a big deal. If you want they could be just passed by value because they are literal types.

To save headache I'll just put it back.